docs/resources/google_compute_zones.md.erb

```diff
@@ -0,0 +1,75 @@
+---
+title: About the google_compute_zones Resource
+platform: gcp
+---
+
+# google\_compute\_zones
+
+Use the `google_compute_zones` InSpec audit resource to test properties of all GCP compute zones for a project in a particular zone.
+
+<br>
+
+## Syntax
+
+A `google_compute_zones` resource block collects GCP zones by project then tests that group.
+
+    describe google_compute_zones(project: 'chef-inspec-gcp') do
+      it { should exist }
+    end
+
+Use this InSpec resource to enumerate IDs then test in-depth using `google_compute_zone`.
+
+    google_compute_zones(project: 'chef-inspec-gcp').zone_names.each do |zone_name|
+      describe google_compute_zone(project: 'chef-inspec-gcp',  zone: zone_name) do
+        it { should exist }
+        its('kind') { should eq "compute#zone" }
+        its('status') { should eq 'UP' }
+      end
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test that there are no more than a specified number of zones available for the project
+
+    describe google_compute_zones(project: 'chef-inspec-gcp') do
```

Comment:
So, similar comments as on `firewalls`.  Use `count` instead of `entries.count`.  You do actually have filter criteria.  Your properties are plural.

Reply:
agreed